suffuse/sfs.go

```diff
@@ -34,7 +38,7 @@ func (u *Sfs) Destroy(ctx context.Context) {
 
 func (u *Sfs) Statfs(ctx context.Context, req *fuse.StatfsRequest, resp *fuse.StatfsResponse) error {
   logD("Statfs")
-  stat, err := u.RootNode.GetPath().SysStatfs()
+  stat, err := RootPath.SysStatfs()
```

Comment:
No, not at all. But neither of them provides meaningful information - I am forwarding statfs from the physical filesystem which happens to underlie a particular path. A fuse filesystem might be on top of multiple physical filesystems or on top of none. This is just to see that it's working.